deploy/awx-operator.yaml

```diff
@@ -635,6 +635,8 @@ spec:
               value: awx-operator
             - name: ANSIBLE_GATHERING
               value: explicit
+            - name: OPERATOR_VERSION
+              value: "0.9.0"
```

Comment:
IHMO should be "devel". Then we should bump this to `expected_bumped_version`  during the release process.
Since we don't have that in place yet, I made it this `0.9.0` for consistency, I would be happy to add the logic so we can have it well placed.